vertx-grpc-protoc-plugin/src/main/java/io/vertx/grpc/protoc/plugin/VertxGrpcGenerator.java

```diff
@@ -21,7 +21,9 @@
 
   private static final int SERVICE_NUMBER_OF_PATHS = 2;
   private static final int METHOD_NUMBER_OF_PATHS = 4;
-  private static final String CLASS_PREFIX = "Vertx";
+
+  private static final String CLASS_PREFIX = System.getProperty("io.vertx.grpc.class.prefix", "Vertx");
```

Comment:
we should use same naming than we use in vertx-core for properties : `vertx.grpc.`